### ISSUE TITLE
Swapping from Gnomish to Goblin Engineering

### ISSUE THREAD

**Issue by Happyfunrock** [Wall-core/Everlook-Bugtracker#725]
I'm not sure that swapping from Gnomish to Goblin Engineering is supposed to be doable in this patch.  It's not looking like it will be right now.  I've started to relevel engineering, and Gnomish Engineering and Artisan Engineering are still in my spellbook even after deleting WDB folder.

Before Deleting WDB:
![image](https://user-images.githubusercontent.com/119549980/204945780-87318b6a-5ed1-4e58-84bc-ac08775df1b9.png)

After Deleting WDB:
![image](https://user-images.githubusercontent.com/119549980/204946099-21e0ff90-b4f0-4ae3-95f9-7e73146c02ad.png)

**Comment by FlagFlayer** [comment]
Yeah, this is an example of https://github.com/vmangos/core/issues/1683. Not sure what causes it or how to fix it as of this moment.

**Comment by Happyfunrock** [comment]
The steps that worked for me were:
1. Delete gnomish engineering card
2. Drop engineering
3.  Exit game and delete WDB folder
4.  Login and then re-pickup engineering
5.  Lvl it to 225, and then pickup breadcrumb quest again

**Comment by FlagFlayer** [comment]
This is due to Book Soothsaying for dummies not being implemented, my vmangos PR should fix this soon
